lib/Elastica/Search.php

```diff
@@ -40,48 +40,41 @@ class Search
     public const OPTION_SEARCH_IGNORE_UNAVAILABLE = 'ignore_unavailable';
 
     /**
-     * @var BuilderInterface
+     * @var BuilderInterface|null
      */
-    private $_builder;
+    private $builder;
 
     /**
-     * Array of indices.
+     * Array of indices names.
      *
-     * @var array
-     */
-    protected $_indices = [];
-
-    /**
-     * Array of types.
-     *
-     * @var array
+     * @var string[]
      */
-    protected $_types = [];
+    protected $indices = [];
```

Comment:
Be aware this part of the renaming could be seen as a breaking change as it is protected.

Reply:
as v7.0 is not yet "released" this could be a good time for it :)